Source/HTTP/WinHttp/winhttp_http_task.cpp

```diff
@@ -1208,6 +1208,7 @@ HRESULT winhttp_http_task::send(
         return HRESULT_FROM_WIN32(dwError);
     }
 
+#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK
```

Comment:
As I mentioned offline, I think this code should be on for GRTS

Reply:
but there's no GRTS `HC_PLATFORM`, right?